pygenie/adapter/genie_3.py

```diff
@@ -283,7 +283,8 @@ def get_info_for_rj(self, job_id, job=False, request=False,
             ret['archive_location'] = data.get('archiveLocation')
             ret['attachments'] = None
             ret['command_args'] = data.get('commandArgs')
-            ret['created'] = data.get('created')
```

Comment:
is the created field being set elsewhere?

Reply:
Not sure how I deleted that.  Possibly I merged the commit wrong.